Makefile

```diff
@@ -133,7 +133,7 @@ AVOCADO_OPTIONAL_PLUGINS_TESTS=$(patsubst %,%/tests/, $(AVOCADO_OPTIONAL_PLUGINS
 endif
 check: clean develop
 	# Unless manually set, this is equivalent to AVOCADO_CHECK_LEVEL=0
-	PYTHON=$(PYTHON) $(PYTHON) -m avocado nrun selftests/*.sh selftests/unit/ selftests/functional/ selftests/jobs/ $(AVOCADO_OPTIONAL_PLUGINS_TESTS)
+	PYTHON=$(PYTHON) $(PYTHON) -m avocado run --test-runner=nrunner --ignore-missing-references -- selftests/*.sh selftests/jobs/* selftests/unit/ selftests/functional/ $(AVOCADO_OPTIONAL_PLUGINS_TESTS)
```

Comment:
Does this mean we will soon drop the nrun command and namespace?

Reply:
Yes, the `nrun` command was supposed to be experimental and temporary.  It was alive for too long.  Now, the point of this PR is that there should be no feature of `avocado nrun` not done better with `avocado run --test-runner=nrunner`, still there are some known voids when compared to the regular runner, that is, `avocado run [--test-runner=runner]`.